code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +414,213 @@ components:
+            and it ensures a consistent level of performance for specific services within the network.
+        maxUpstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort data
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxUpstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The burst rate for best effort data
+        targetMinDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            This is the target minimum downstream rate for the QoS profile.
+
+            For 5G networks 3GPP Guaranteed Bit Rate (GBR) refers to a dedicated, fixed data rate assigned to
+            specific services, ensuring a minimum performance level. As per 3GPP TS 23.203,
+            GBR is a QoS parameter used to manage traffic classes in mobile networks. It
+            provides a stable data rate for latency-sensitive applications, such as voice calls or
+            video streaming, helping maintain a consistent user experience.
+
+            The committed data rate allocated to specific services, ensuring a consistent level of
+            performance. DOCSIS networks, the "Minimum Reserved Traffic Rate" is defined in the 
+            DOCSIS 3.1 - MAC and Upper Layer Protocols Interface Specification" 
+            and it ensures a consistent level of performance for specific services within the network.
+        maxDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort rate
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxDownstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort burst rate
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        minDuration:
+          $ref: "#/components/schemas/Duration"
+          description: The shortest time period that is supported this profile can be deployed.
+        maxDuration:
+          $ref: "#/components/schemas/Duration"
+          description: The maximum time period that this profile can be deployed.
+        priority:
+          type: number
+          example: 2.0
+          description: |
+            Priority levels allow efficient resource allocation and ensure optimal performance
+            for various services in each technology, with the highest priority traffic receiving
+            preferential treatment.
+            The lower value the higher priority.
+            Since not all access networks use the same priority range, so this priority will be
+            scaled to the access network's priority range.
+          format: float
+          minimum: 1
+          maximum: 10
+        packetDelayBudget:
+          $ref: "#/components/schemas/Duration"
+          description: |
```

Comment:
This description is too 3GPP centric. And it also doesn't match the 3GPP definition, which is the maximum one-way latency between the device and the core network, so excludes the path outside of the carrier's network. So it is not a commitment to maintain the end-to-end latency below this value.

Reply:
I'll remove the end-to-end reference.